### ISSUE TITLE
Please update README: For Filemaker 19 and up

### ISSUE THREAD

**Issue by karstenw** [bc-m/fm-xml-export-exploder#1]
It took me an hour as a nonrust programmer ( but able to compile & run ) why fm-xml-export-exploder silently did nothing on a XML saved by FMP18.

**Comment by karstenw** [comment]
Yep. The FMP18 structure is a little different. The root element is named "FMDynamicTemplate".

**Comment by karstenw** [comment]
> Bonus: You can also use prebuilt executables from release page instead compile by yourself: https://github.com/BC-M/fm-xml-export-exploder/releases/latest

To debug I learned a lot about println!()

**Comment by BC-M** [comment]
Thanks for your reply. I will test it with FM18. If the structure is too different, I will update the readme.

**Comment by karstenw** [comment]
FMP18 XMLSaveAs structure

```
FMDynamicTemplate
  Structure
    AddAction
      BaseDirectoryCatalog
        BaseDirectory
      FileAccessCatalog
        UUID
        ObjectList
          Authorization
      BaseTableCatalog
        BaseTable
      TableOccurrenceCatalog
        TableOccurrenceNotes
        TableOccurrence
      CustomFunctionsCatalog
        ObjectList
          CustomFunction
      FieldsForTables
        FieldCatalog
          TableOccurrenceReference
          ObjectList
      ValueListCatalog
        ValueList
      RelationshipCatalog
        Relationship
          LeftTable
          RightTable
          JoinPredicateList
      CalcsForCustomFunctions
        ObjectList
          CustomFunctionCalc
      ScriptCatalog
        Script
      ThemeCatalog
        Theme
      LayoutCatalog
        Layout
      PrivilegeSetsCatalog
        ObjectList
          PrivilegeSet
      ExtendedPrivilegesCatalog
        ObjectList
          ExtendedPrivilege
      AccountsCatalog
        Options
        ObjectList
          Account
      StepsForScripts
        Script
          ScriptReference
          ObjectList
            Step
      CustomMenuCatalog
        PasteIndexList
        CustomMenu
          MenuItemList
            CustomMenuItem
          Conditions
            Install
      CustomMenuSetCatalog
        PasteIndexList
        CustomMenuSetReference
        ObjectList
          CustomMenuSet
    ModifyAction
      FieldsForTables
  Metadata
    AddAction
      Minimum
      Login
      SavePassword
      Spelling
      Text
      ScriptTriggers
```

**Comment by bc-m** [comment]
@karstenw I just released [v0.1.1](https://github.com/bc-m/fm-xml-export-exploder/releases/tag/v0.1.1) with support for FileMaker 18.
